internal/builders/go/README.md

```diff
@@ -101,16 +101,16 @@ We think gradual adoption is good for projects to get used to SLSA.
 
 The configuration file accepts many of the common fields Goreleaser uses, as you can see in the [example](#configuration-file). The configuration file also supports two variables: `{{ .Os }}` and `{{ .Arch }}`. Other variables can be set manually as shows in the table below, in combination with the builder's `evaluated-envs`:
 
-| Name                 | Value                                                                                                                            | Example                                    |
-| -------------------- | -------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
-| `{{ .CommitDate }}`  | `date -d @$(git log --date=iso8601-strict -1 --pretty=%ct)`                                                                      | `Mon Jun 13 01:23:36 AM UTC 2022`          |
-| `{{ .FullCommit }}`  | `$GITHUB_SHA` or `$(git rev-parse HEAD)`                                                                                         | `b2a980888f359b8cef22cb61f153746e1a06deb0` |
-| `{{ .ShortCommit }}` | `$(echo $GITHUB_SHA \| cut -c1-8)` or `$(git rev-parse HEAD \| cut -c1-8)`                                                       | `b2a98088`                                 |
-| `{{ .Version }}`     | `$(git describe --tags --always --dirty \| cut -c2-)` or `$(echo $GITHUB_REF_NAME \| cut -c2-)` on new tags and release triggers | `1.2.3-alpha+b2a98088`                     |
-| `{{ .Tag }}`         | `$GITHUB_REF_NAME` (on `release` and `push` new tag triggers) or `$(git describe --tags --always --dirty \| cut -c2-)`           | `v1.2.3-alpha+b2a98088`                    |
-| `{{ .Major }}`       | `$(git describe --tags --always --dirty \| cut -d '.' -f1 \| cut -c2-)`                                                          | `1`                                        |
-| `{{ .Minor }}`       | `$(git describe --tags --always --dirty \| cut -d '.' -f2`                                                                       | `2`                                        |
-| `{{ .Patch }}`       | `$(git describe --tags --always --dirty \| cut -d '.' -f3 \| cut -d '-' -f1 \| cut -d '+' -f1`                                   | `3`                                        |
+| Name                 | Value                                                                                                      | Example                                    |
+|----------------------|------------------------------------------------------------------------------------------------------------|--------------------------------------------|
```

Comment:
those changes seem to be breaking the table

Reply:
Took care of it. Thanks